snippets/language-html.cson

```diff
@@ -110,6 +110,9 @@
     'prefix': 'embed'
     'body': '<embed type="${1:video/quicktime}" src="${2:#}" width="${3:300}" height="${4:300}">$0'
   # F
+  'Favicon':
+    'prefix': 'favicon'
+    'body': '<link rel="shortcut icon" href="$1.ico" />$0'
```

Comment:
Other self-closing tags in this file seem to use just `>` instead of ` />`.  To fit in more I would suggest `<link rel="shortcut icon" href="$1.ico">$0`.